src/library_webgpu.js

```diff
@@ -2686,6 +2689,11 @@ var LibraryWebGPU = {
     var context = WebGPU.mgrSurface.get(surfaceId);
 
 #if ASSERTIONS
+    var viewFormatCount = {{{ gpu.makeGetU32('config', C_STRUCTS.WGPUSurfaceConfiguration.viewFormatCount) }}};
+    var viewFormats = {{{ makeGetValue('config', C_STRUCTS.WGPUSurfaceConfiguration.viewFormats, '*') }}};
+    assert(viewFormatCount === 0 && viewFormats === 0, "TODO: Support viewFormats.");
+    var alphaMode = {{{ gpu.makeGetU32('config', C_STRUCTS.WGPUSurfaceConfiguration.alphaMode) }}};
+    assert({{{ gpu.CompositeAlphaMode.Opaque }}} === alphaMode, "TODO: Support alphaMode.");
```

Comment:
Ah... we should allow `Auto` as well, since it is the default in the C++ header, and it always does the same thing as `Opaque` in Emscripten anyway. [1]

```suggestion
    assert(alphaMode === {{{ gpu.CompositeAlphaMode.Auto }}} ||
      alphaMode === {{{ gpu.CompositeAlphaMode.Opaque }}},
      "TODO: Support WGPUCompositeAlphaMode_Premultiplied.");
```

Reply:
FYI The default is "Opaque" not "Auto" in https://source.chromium.org/chromium/chromium/src/+/main:out/linux-Debug/gen/third_party/dawn/include/dawn/webgpu_cpp.h;l=2965;drc=6443d1ded6b6a1b62c65099d5014ab240b68946f

Reply:
I've started https://dawn-review.googlesource.com/c/dawn/+/190080 to address this issue and will update Emscripten.